examples/open_loop_control_fetch/scene_config.kujiale_0000.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+SIMULATION_CONTROLLER:
+  URDFBOT_AGENT:
+    SPAWN_MODE: "specify_pose" # "specify_pose", "specify_existing_actor"
+    SPAWN_ACTOR_NAME: "PlayerStart"
```

Comment:
If SPAWN_MODE is specify_pose, then SPAWN_ACTOR_NAME has no effect, so let's keep it set to the empty string (or remove it entirely) to avoid confusion.

Reply:
fixed